piwik-sdk/src/main/AndroidManifest.xml

```diff
@@ -4,4 +4,14 @@
 
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+
+    <application>
+        <receiver
+            android:name=".InstallReferrerReceiver"
```

Comment:
So the dev is aware that Piwik registers a broadcastreceiver? Or in case the dev wants to register this receiver himself? We could rebroadcast the intent to allow registering multiple receivers for this intent.